Fix visibility of the close button in status card

lovery opened the pull request:
The close button is not visible when there is an image for the device and the theme of the phone is dark, but it is there and you can click on it if you know where exactly it is:
![IMG_20251027_140623](https://github.com/user-attachments/assets/cbcc215e-83a0-43c8-9d4c-d0ba12b6387e)

I've added a line with the device name and the close button like it is when there is no image.

This behavior is happening on Pixel 7 and Pixel 8 with Chrome and Firefox.

tananaev commented:
We have this style on the icon:
```
  mediaButton: {
    color: theme.palette.primary.contrastText,
    mixBlendMode: 'difference',
  },
```
Which should make it visible, but I guess it doesn't work for some reason. So that part needs fixing.

lovery committed:
Fix visibility of the close button in status card

Add a semi-transperant background color to the icon's container
and change the color of the close icon.

lovery commented:
Here is a few screenshots of how the fix looks like:
<img width="100px" max-height="1061px" alt="Screenshot from 2025-10-28 13-28-28" src="https://github.com/user-attachments/assets/7ef98271-ee91-4153-bceb-0053723f57d9" /><img width="100" height="1061" alt="Screenshot from 2025-10-28 13-27-01" src="https://github.com/user-attachments/assets/a2a8b965-225e-4e73-9529-5e7458b693d3" /><img width="100" height="1061" src="https://github.com/user-attachments/assets/24278333-c0d3-4ef9-9d5c-3a77bd2f6490"/><img width="100" height="1061" src="https://github.com/user-attachments/assets/ebec5270-67ed-4d58-a4f7-696483fa72ac"/><img width="100" height="1061" src="https://github.com/user-attachments/assets/166aa250-2452-466c-9124-a25afbb807c6"/>

tananaev commented:
I already pointed it out earlier. `mix-blend-mode: difference;` should inverse the color, so it should be always visible, no matter what the background is.

tananaev commented:
Why is it unreasonable?
